configs/ote_custom_classification/mobilenet_v3_large_1/template.yaml

```diff
@@ -0,0 +1,46 @@
+# Description.
+name: MobileNet-V3-large-1x
+model_template_id: Custom_Image_Classification_MobileNet-V3-large-1x
+task_type: CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Custom Image Classification MobileNet-V3-large-1x
+
+# Algo backend.
+# Would be better to have something more precise and self-contained, e.g.:  docker image or path to Python venv activation script?
+framework: OTEClassification v1.2.3
+
+# Should be revisited?
+application:
+  ~
+
+# # Optional? requirements to the dataset. Should be revisited?
+# dataset_requirements:
+#   ~
+
+# Task implementation.
+entrypoints:
+  base: torchreid.integration.sc.task.OTEClassificationTask
+  openvino: torchreid.integration.sc.openvino_task.OpenVINOClassificationTask
+
+# Task and hyperparameters.
+hyper_parameters:
+  base_path: "../../../torchreid/integration/sc/configuration.yaml"
+  parameter_overrides:
+    learning_parameters:
+      batch_size:
+        default_value: 30
+      max_num_epochs:
+        default_value: 200
+      learning_rate:
+        default_value: 0.001
```

Comment:
This default lr will destroy multilabel classification, unfortunately. But I will try to change the  ASL loss value to fit bigger learning rate values.

Reply:
Perhaps multiplying ASL by 0.1 could help to align it's magnitude with SM.